### PR TITLE
[COST-3554] Calculate and store AWS amortized cost

### PR DESCRIPTION
**Pull request by samdoran**
## Jira Ticket

[COST-3554](https://issues.redhat.com/browse/COST-3554)

## Description

Calculates AWS amortized cost based on LineItemType and unblended cost, falling back to savings plan effective cost.

## Testing

TBD

## Notes

- [ ] Use `calculated_amortized_cost` in UI code instead of `savingsplan_savingsplaneffectivecost`

### CODE REVIEW

File: koku/masu/processor/aws/aws_report_parquet_processor.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def __init__(self, manifest_id, account, s3_path, provider_uuid, parquet_local_p
             "lineitem_unblendedrate",
             "lineitem_blendedcost",
             "lineitem_blendedrate",
-            "savingsplan_savingsplaneffectivecost",
+            "calculated_amortized_cost",
```
Comment (adberglund) [inline koku/masu/processor/aws/aws_report_parquet_processor.py:29]: We want to keep the `savingsplan_savingsplaneffectivecost` as the calculated value is just being sent to Postgres, and not in Trino.

File: koku/masu/util/aws/common.py [hunk 2]
```diff
@@ -646,7 +646,7 @@ def aws_generate_daily_data(data_frame):
             "lineitem_blendedcost": ["sum"],
             "pricing_publicondemandcost": ["sum"],
             "pricing_publicondemandrate": ["max"],
-            "savingsplan_savingsplaneffectivecost": ["sum"],
+            "calculated_amortized_cost": ["sum"],
```
Comment (adberglund) [inline koku/masu/util/aws/common.py:649]: Here we don't want to make a change, as the calculated cost doesn't exist in the raw data. This function just rolls up raw hourly data to daily data.

**Review by adberglund — CHANGES_REQUESTED** [review]

File: koku/masu/util/aws/common.py (hunk 2 above)
Comment (samdoran) [reply koku/masu/util/aws/common.py:649]: Ok, so where do we need to return `calculated_amortized_cost` then?

**Comment by samdoran** [comment]
It looks like this needs to be changed in the UI:

https://github.com/project-koku/koku-ui/blob/88ab53d4ed02987116fbbb940f947ba1f9adf7b4/src/routes/views/components/costType/costType.tsx#L44

I think the `AWSProviderMap` will then return the correct value from the database.

**Review by adberglund — APPROVED** [review]

**Pull request merged**